.pre-commit-config.yaml.jinja

```diff
@@ -1,3 +1,30 @@
+{#- This namespace holds repo versions; other variable definitions are found
```

Comment:
@Yajo Would it be more readable to duplicate the full definition of the pre-commit hooks? Personally, I find that the conditional inline addition/removal of hooks is not very readable. :thinking:

Reply:
I have encountered feelings about that.

On one side, indeed the configs would be more readable.

On the other side, maintenance is easier. For example,  #9 would be added in a single file, and both v13 and v14 repos would benefit from that enhancement.

Although reading all those "ifs", in reality I can see that from the time we delivered v13 branches, the main diff is that all projects we use started supporting pre-commit out of the box (except pylint-odoo with https://github.com/OCA/pylint-odoo/issues/296, which would be nice if fixed before v14 creation so we can remove that hack from here). So, we can expect less diffs the next year; just the versions probably.

Then, maybe I can create one for v13 only, and other one for any other version. If for v15 we see a lot of diff, we do the same. Otherwise, we just update versions. WDYT?

Reply:
My instinct would also have been to create a template branch for 13.0, 14.0, etc but I guess that ship has sailed, so let's see where it goes.

Reply:
Kudos for not following your instinct! :blush: 

We have a templating engine for some reason. :wink: 

I have applied your suggestions. I hope you find it more readable now.